packages/launchpad/src/error/BaseError.vue

```diff
@@ -1,50 +1,51 @@
 <template>
-  <div class="mx-auto space-y-32px text-center min-w-476px max-w-848px pt-16px children:text-center">
+  <div
+    v-if="baseError"
+    class="mx-auto space-y-32px text-center min-w-476px max-w-848px pt-16px children:text-center"
+  >
     <div>
       <h1
-        class="font-medium leading-snug text-32px text-gray-900"
+        class="font-medium leading-snug pb-24px text-32px text-gray-900"
         data-testid="error-header"
       >
         <slot name="header">
-          {{ headerText }}
+          {{ baseError.title }}
         </slot>
       </h1>
       <!-- eslint-disable vue/multiline-html-element-content-newline  -->
 
       <slot name="message">
```

Comment:
Pass in your component with the props here and we should be good

`<OpenInIDE>`

Reply:
https://github.com/cypress-io/cypress/pull/19872/commits/dd1ee9af037b56e0aa1d095c0a7a6d5745ad1051